docker/node/Dockerfile

```diff
@@ -6,7 +6,6 @@ WORKDIR /app
 
 RUN apt-get update && apt-get --no-install-recommends install -y \
     bzip2 \
-    git \
```

Comment:
``git`` may have been needed by earlier versions of the build process, but does not appear to be needed now. ``bzip2`` is still needed to unpack ``phantoms``, and ``make`` is used to create the bundled assets.